src/librustc/infer/higher_ranked/mod.rs

```diff
@@ -593,7 +593,7 @@ impl<'a, 'gcx, 'tcx> InferCtxt<'a, 'gcx, 'tcx> {
     where
         T : TypeFoldable<'tcx>,
     {
-        let new_universe = self.create_subuniverse();
+        let new_universe = self.create_superuniverse();
```

Comment:
I'm not sure I prefer "superuniverse" to "subuniverse". Universes naturally form a tree, and the "subelement" relationship goes from a universe to its sub/superuniverse.